### ISSUE TITLE
snakemake version problem?

### ISSUE THREAD

**Issue by flowers9** [aaronphillips7493/CLAW#3]
I just did a new install of CLAW (and required other packages), and I'm getting an error trying to run the example:

```
#snakemake --profile profiles/slurm --use-conda --keep-going

snakemake: error: unrecognized arguments: --cluster-status=profiles/slurm/status --cluster-config=cluster-configs/slurm.yaml
```

The snakemake version is 8.5.4 (which is current), and I can't find any reference to those options in the current docs (or the snakemake code).  When I looked through the snakemake change logs (https://snakemake.readthedocs.io/en/stable/project_info/history.html), I can find references to them in versions 3.7.0 and 4.1.0.

Is there a particular version (or range of versions) of snakemake that's required for CLAW?  Or is my problem something else enitrely?

The only changes I made to the install was adding a default value to cluster.account and adding a cluster.licenses field (required for the slurm cluster I use).

**Comment by flowers9** [comment]
Yep, looks like snakemake 7.32.4 is the last version that supports CLAW.  They broke a lot of stuff in the jump to 8.0.0.  Maybe mention it somewhere?

**Comment by dreyes17** [comment]
Hello,

As far as I have been able to verify, it only works with version 7.30.1, not with 7.32.4.

This is the Dockerfile I have been able to create for it to work correctly. The only prerequisite is to download the CLAW code, rename it from CLAW-main to CLAW, and compress it into a .tar.xz file. This file should be in the same folder as the Dockerfile.

```Dockerfile
FROM snakemake/snakemake:v7.30.1

WORKDIR /app

COPY CLAW.tar.xz /app/

RUN tar -xf CLAW.tar.xz

WORKDIR /app/CLAW

RUN conda config --set allow_conda_downgrades true

RUN conda config --set auto_update_conda false

RUN conda config --add channels conda-forge

RUN conda config --add channels bioconda

RUN conda create --name snakemake

RUN conda init bash

RUN ["/bin/bash", "-c", "source ~/.bashrc"]

RUN ["/bin/bash", "-c", "conda install mamba -n base -c conda-forge"]

RUN ["/bin/bash", "-c", "mamba install snakemake==7.30.1"]

RUN ["/bin/bash", "-c", "mamba install biopython"]

RUN ["/bin/bash", "-c", "snakemake -j 1 --conda-create-envs-only --use-conda"]

CMD ["tail", "-f", "/dev/null"]
```

**Comment by Bank-tidy** [comment]
> I just did a new install of CLAW (and required other packages), and I'm getting an error trying to run the example:
> 
> ```
> #snakemake --profile profiles/slurm --use-conda --keep-going
> 
> snakemake: error: unrecognized arguments: --cluster-status=profiles/slurm/status --cluster-config=cluster-configs/slurm.yaml
> ```
> 
> The snakemake version is 8.5.4 (which is current), and I can't find any reference to those options in the current docs (or the snakemake code). When I looked through the snakemake change logs (https://snakemake.readthedocs.io/en/stable/project_info/history.html), I can find references to them in versions 3.7.0 and 4.1.0.
> 
> Is there a particular version (or range of versions) of snakemake that's required for CLAW? Or is my problem something else enitrely?
> 
> The only changes I made to the install was adding a default value to cluster.account and adding a cluster.licenses field (required for the slurm cluster I use).

Hello, could you please share your packaged docker file? I still get an error when I run it，thank you very much !
